llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -37,6 +37,15 @@ class RISCVExtension<string name, int major, int minor, string desc,
   bit Experimental = false;
 }
 
+// The groupID/bitmask of RISCVExtension is used to retrieve a specific bit value 
+// from __riscv_feature_bit based on the groupID and bitmask.
+// gourpID      - groupID of extension
+// bitmaskShift - bitmask shift of extension
+class RISCVExtensionBitmask<bits<3> groupID, int bitmaskShift> {
```

Comment:
Can we merge groupID and bitmask shift into a single value where 0-63 is group 0, 64-127 is group 1, 128-191 is group 3, etc.

Reply:
The RFC side defines the bit position in GroupID/bitmask form. Is it worth merging into a single value instead of aligning with the RFC?

> Note: When building the required extension bitmasks from a multiversion function, we still need to break the single value into GroupID/bitmask.